### PR TITLE
test event

### PR DESCRIPTION
**Pull request by shenxianpeng**

### CODE REVIEW

**Review by 2bndy5 — APPROVED** [review]

**Comment by 2bndy5** [comment]
I'm not sure if there's anything we can do about removing thread comments from before the PR status change (opened ▶️ closed ▶️ reopened).

**Comment by shenxianpeng** [comment]
It seems OK because this is not happening a lot, and the previous thread comment can be used to compare with the new thread comment.

File: .github/workflows/test-cpp-linter.yml [hunk 1]
```diff
@@ -1,18 +1,22 @@
...
   pull_request:
     types: [opened, reopened]  # let PR-synchronize events be handled by push events
   push:
   workflow_dispatch:
-
+    inputs:
+      branch_name:
+        description: 'The test branch name of cpp-linter-action'
+        required: true
+        default: 'master'
```
Comment (2bndy5) [inline .github/workflows/test-cpp-linter.yml:12]: This is an interesting idea! I like it.

Comment (shenxianpeng) [reply .github/workflows/test-cpp-linter.yml:12]: but it failed. I don't have much time to fix it tonight because I need to take care of the new baby :(

Comment (2bndy5) [reply .github/workflows/test-cpp-linter.yml:12]: Oh, congrats on the family member! No worries here. I'm a little distracted (getting the entire [nRF24 org](https://www.github.com/nRF24) setup to use our action). I'll probably have a look tomorrow.

Comment (shenxianpeng) [reply .github/workflows/test-cpp-linter.yml:12]: THANK YOU!

File: .github/workflows/test-cpp-linter.yml [hunk 2]
```diff
@@ -16,7 +16,7 @@ jobs:
     runs-on: ubuntu-latest
     steps:
       - uses: actions/checkout@v2
-      - uses: cpp-linter/cpp-linter-action@"${{ github.event.inputs.branch_name }}"
+      - uses: cpp-linter/cpp-linter-action@${{ github.event.inputs.branch_name }}
```
Comment (2bndy5) [inline .github/workflows/test-cpp-linter.yml]: I vaguely remember trying this in #10 development, but it didn't work for me. I don't think the `uses` option accepts/allows dynamic variables.

Comment (shenxianpeng) [reply .github/workflows/test-cpp-linter.yml]: En, it looks like `users` option doesn't allowed dynamic variables

**Pull request closed without merging**